.github/workflows/nix.yml

```diff
@@ -0,0 +1,23 @@
+name: nix
+
+on:
+  pull_request:
+    paths:
+      - Cargo.lock
+      - "*.nix"
+      - justfile
+      - .github/workflows/nix.yml
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Nix
+      uses: cachix/install-nix-action@v12
```

Comment:
Please use the full SHA versions of these actions, as we do in other workflows.  This prevents the tag from changing without a PR (which dependabot will do as these actions are updated).